stdlib/system/src/main/ballerina/system/system.bal

```diff
@@ -40,3 +40,71 @@ public function getUserHome() returns string = external;
 #
 # + return - The random string
 public function uuid() returns string = external;
+
+# Reports whether file or directory exists for the given the
+# path.
+#
+# + path - String value of file path.
+# + return - True if path is absolute, else false
+public function exists(string path) returns boolean = external;
+
+# Creates a new directory with the specified file name.
```

Comment:
It is better if we can wrap at 120 characters to be consistent with the code formatting.